docs/pages/versions/unversioned/sdk/sqlite-next.mdx

```diff
@@ -327,10 +327,13 @@ async function openDatabase(pathToDatabaseFile: string): Promise<SQLite.Database
   if (!(await FileSystem.getInfoAsync(FileSystem.documentDirectory + 'SQLite')).exists) {
     await FileSystem.makeDirectoryAsync(FileSystem.documentDirectory + 'SQLite');
   }
-  await FileSystem.downloadAsync(
-    Asset.fromModule(require(pathToDatabaseFile)).uri,
-    FileSystem.documentDirectory + 'SQLite/myDatabaseName.db'
-  );
+  const localDbPath = FileSystem.documentDirectory + 'SQLite/myDatabaseName.db';
+  const asset = Asset.fromModule(require(pathToDatabaseFile));
+  if (asset.localUri) {
+    await FileSystem.copyAsync({ from: asset.localUri, to: localDbPath });
+  } else {
+    await FileSystem.downloadAsync(asset.uri, localDbPath);
+  }
```

Comment:
Another way to write this that I think would work (but I haven't tested it) is to use [Asset.downloadAsync](https://docs.expo.dev/versions/latest/sdk/asset/#downloadasync):
```suggestion
  const asset = await Asset.fromModule(require(pathToDatabaseFile)).downloadAsync();
  await FileSystem.copyAsync({ from: asset!.localUri, to: localDbPath });
```

Reply:
i can confirm that the null Asset.uri is not really a regression on sdk-50. on sdk-49, even it has Asset.uri but the uri cannot use FileSystem.downloadAsync() to download. the old doc might work for classic updates but not eas update.

i can also confirm the `Asset.downloadAsync()` work. even it requires an extra copy for embedded asset, but making the doc clearer. let's use this approach. thanks @ide for the suggestion.